vars/jobs.groovy

```diff
@@ -38,3 +38,16 @@ def build_job(name, parameters, is_stable)
   }
   return job
 }
+
+def executor(name, parameters, is_runner=false){
+  def current_job = build job: name, parameters: parameters, propagate: false, wait: true
+  if (current_job.result != 'SUCCESS') {
+    if (env.NODE_NAME.contains('flow')) {
+      notification.notificator(env.BUILD_URL, get_job_url(current_job))
+    }
+    if(is_runner) {
+      throw new Exception(current_job.result)
+    }
```

Comment:
If you are throwing general exception in both cases why not throw only one exception like 

`throw new Exception(other.parse_url(get_job_url(current_job), true) + " has FAILED with status: ${get_job_url(current_job)}")`